disassemblers/ofrak_ghidra/server.conf

```
@@ -156,7 +156,10 @@ ghidra.repositories.dir=./repositories
 #                       repositories directory must be retained.
 wrapper.app.parameter.1=-a0
 wrapper.app.parameter.2=-e0
-wrapper.app.parameter.3=${ghidra.repositories.dir}
+wrapper.app.parameter.3=-u
+wrapper.app.parameter.4=-ip
+wrapper.app.parameter.5=0.0.0.0
```

```suggestion
wrapper.app.parameter.4=-i0.0.0.0
```

Could also use `-iphostname`